rebasehelper/tests/functional/test_rebase.py

```diff
@@ -86,6 +86,9 @@ def cloned_dist_git(self, url, commit, workdir):
     ])
     @pytest.mark.usefixtures('cloned_dist_git')
     def test_rebase(self, buildtool, package, version, patches):
+        if buildtool == 'rpmbuild' and os.geteuid() != 0:
+            pytest.skip('rpmbuild needs root permission.')
+
```

Comment:
You could do the same thing this way (and I like it a bit better):

```python
     @pytest.mark.parametrize('buildtool', [
         pytest.mark.skipif(
             os.geteuid() != 0,
             reason='requires superuser privileges')
         ('rpmbuild'),
         'mock',
     ])
```

Reply:
ok, just moment. I will fix it.